### PR TITLE
Stash V1beta1 E2E test for StatefulSet

### PR DESCRIPTION
**Pull request by suaas21**
xref: #707

Tasks:
- [x] Workload backup and restore for Statefulset

### CODE REVIEW

File: test/e2e/framework/statefulset.go [hunk 1]
```diff
@@ -33,6 +35,69 @@ func (fi *Invocation) StatefulSet(pvcName string) apps.StatefulSet {
 	}
 }
 
+func (fi *Invocation) StatefulSetv1beta1() apps.StatefulSet {
```
Comment (hossainemruz) [inline test/e2e/framework/statefulset.go]: Rename `StatefulSetv1beta1()` to `StatefulSetForV1beta1API()`

File: test/e2e/workload_test.go [hunk 2]
```diff
@@ -303,3 +320,311 @@ var _ = Describe("Deployment", func() {
 		})
 	})
 })
+
+var _ = Describe("Statefulset", func() {
+	var (
```
Comment (hossainemruz) [inline test/e2e/workload_test.go]: Why repeat these variables? Reuse them.

File: test/e2e/workload_test.go [hunk 3]
```diff
@@ -303,3 +320,311 @@ var _ = Describe("Deployment", func() {
...
+			restoredData, err = f.ReadSampleDataFromMountedDirectory(ss.ObjectMeta, framework.GetPathsFromRestoreSession(&restoreSession), apis.KindStatefulSet)
+			Expect(err).NotTo(HaveOccurred())
+
+			By("Compare between restore data and sample data")
+			Expect(restoredData).To(BeEquivalentTo(sampleData))
+
+		})
+	})
+
+	Context("Backup && Restore data on different StatefulSet", func() {
```
Comment (hossainemruz) [inline test/e2e/workload_test.go]: `Backup && Restore data on different StatefulSet`
remove `Backup &&`

File: test/e2e/workload_test.go (hunk 2 above)
Comment (suaas21) [reply test/e2e/workload_test.go]: fixed

File: test/e2e/workload_test.go (hunk 3 above)
Comment (suaas21) [reply test/e2e/workload_test.go]: fixed

**Pull request merged**